lib/string.js

```diff
@@ -70,6 +70,13 @@ internals.String.prototype._base = function (value, state, options) {
         if (this._flags.trim) {
             value = value.trim();
         }
+
```

Comment:
This should probably live outside of convert since it's not a test, modify the previous if to keep it DRY.